packages/core/src/version.ts

```diff
@@ -20,6 +20,15 @@ export function getVersion(bytecode: string, linkedBytecode?: string): Version {
 }
 
 export function hashBytecode(bytecode: string): string {
```

Comment:
Although I think we should ship this fix as soon as possible, can we remove it from this PR so we keep this one about overrides only?

Reply:
The fix is needed for the tests to work correctly